### PR TITLE
[JENKINS-68484] Users views icon is misleading

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-68484](https://issues.jenkins.io/browse/JENKINS-68484).

### Before
Before

### After
After

### Proposed changelog entries

* Use glasses icon to replace user icon in user page

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7001"></a>

### CODE REVIEW

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (NotMyFault) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:518]:
```suggestion
```

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (NotMyFault) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:601]:
```suggestion
```

**Review by NotMyFault — COMMENTED** [review]
Icon translation is redundant for symbols that don't exist as Tango icons.

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (Langer0416) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java]: removed it, thanks

**Comment by daniel-beck** [comment]
Is that really a (more) appropriate icon?

Is there perhaps a "tab" like icon? A rectangle with a tab-shaped protrusion on top?

**Comment by Langer0416** [comment]
> Is that really a (more) appropriate icon?
> 
Maybe we can wait for more maintainers to discuss this
> Is there perhaps a "tab" like icon? A rectangle with a tab-shaped protrusion on top?
>
Do you mean the icon like this?
icon

**Comment by daniel-beck** [comment]
> Do you mean the icon like this?

That might work.

The limitations of using an icon set that's clearly designed for mobile apps…

**Comment by Langer0416** [comment]
I have made some modifications, please have a look at it.
### After
After

**Review by uhafner — APPROVED** [review]
Please update the screenshot in the description.

**Comment by Langer0416** [comment]
> Please update the screenshot in the description.

I've updated it, thanks for reviewing

**Review by daniel-beck — APPROVED** [review]

**Comment by NotMyFault** [comment]
The symbol chosen misses a color attribute. I went ahead and fixed that on this PR to prevent a future regression:

Screenshot 2022-08-20 at 13 47 28

vs

Screenshot 2022-08-20 at 13 50 54

**Review by NotMyFault — APPROVED** [review]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Comment by uhafner** [comment]
It would be helpful if we can document this color problem (or fix it with a script?), so we do not have that problem again and again when we migrate additional icons. I still think it would be helpful if we integrate all those icons in one step into core somehow.

**Comment by NotMyFault** [comment]
> It would be helpful if we can document this color problem

It's part of the symbols page in the design library, outlining which attributes a symbol has to have in order to be theme-able. In the past, I filed PRs to ionicon fixing these issues, none of them have been pulled yet, FTR.

> integrate all those icons in one step into core

+1, ionicons exist as npm package, e.g. 👀

**Pull request merged**
